secrets/dataplane-observability-config.yaml

```diff
@@ -0,0 +1,17 @@
+---
+# the url points to an auth proxy before the remote write endpoint
+# this is optional, either proxy or direct configurations must be provided
+proxy:
+  remoteWriteUrl: https://observability-rw-proxy-managed-services-stage.apps.app-sre-stage-0.k3s7.p1.openshiftapps.com
+
+# the url points directly to an (authenticated) remote write endpoint
+# this is optional, either proxy or direct configurations must be provided
+# direct:
+#   remoteWriteUrl: https://observatorium-mst.api.stage.openshift.com
+#   metricsClientId: dummy
+#   metricsClientSecret: dummy
+
+# optional: a GitHub token to pull observability resources
+# if not provided, the resources have to be either a public GitHub repository, or
+# a public quay repository
```

Comment:
It would have been nice to also being able to be explicit about this but OO just infers that for now.